.github/workflows/aws.yml

```
@@ -35,7 +35,7 @@ env:
   ECR_REPOSITORY: obtoo-registry           # set this to your Amazon ECR repository name
   ECS_SERVICE: obtoo-task-service-jufgal33                 # set this to your Amazon ECS service name
   ECS_CLUSTER: oubtoo-cluster                 # set this to your Amazon ECS cluster name
-  ECS_TASK_DEFINITION: .aws/obtoo-task-revision1.json # set this to the path to your Amazon ECS task definition
+  ECS_TASK_DEFINITION: .aws/obtoo-task-revision4.json # set this to the path to your Amazon ECS task definition
```

YAMLlint (1.37.1)

[warning] too few spaces before comment: expected 2

(comments)